docker/cygnus-twitter/Dockerfile

```diff
@@ -43,18 +44,19 @@ ENV FLUME_TGZ "apache-flume-${FLUME_VER}-bin.tar.gz"
 ENV FLUME_URL "http://archive.apache.org/dist/flume/${FLUME_VER}/${FLUME_TGZ}"
 ENV FLUME_HOME "/opt/apache-flume"
 
-ENV JAVA_HOME /usr/lib/jvm/java-1.6.0
+ENV JAVA_VERSION=1.7.0
 
 # Add Cygnus user
 RUN adduser ${CYGNUS_USER}
 
 # Install
-RUN yum -y install tar nc which git java-1.6.0-openjdk-devel && \
+RUN yum -y install tar nc which git java-${JAVA_VERSION}-openjdk-devel && \
     yum clean all && \
+    export JAVA_HOME=/usr/lib/jvm/java-${JAVA_VERSION} && \
     curl --remote-name --location --insecure --silent --show-error "${MVN_URL}" && \
     tar xzvf ${MVN_TGZ} && \
     mv apache-maven-${MVN_VER} ${MVN_HOME} && \
-    export MAVEN_OPTS="-Xmx512m -XX:MaxPermSize=128m" && \
+    export MAVEN_OPTS="-Xmx512m -XX:MaxPermSize=128m -Dfile.encoding=UTF-8 -Dproject.build.sourceEncoding=UTF-8" && \
```

Comment:
As discussed offline, I think this wont affect the JVM but only Maven.

`-Dfile.encoding=UTF-8` must be added to the starting command within `cygnus-entrypoint.sh` instead.